lms/djangoapps/grades/signals/handlers.py

```diff
@@ -133,6 +133,15 @@ def enqueue_subsection_update(sender, **kwargs):  # pylint: disable=unused-argum
             kwargs.get('points_possible'),
         )
     )
+    log.info(
+        u'Grades: Request async calculation of subsection grades. '
+        u'user_id:{}, course_key:{}, usage_key:{}, Task id: {}'.format(
+            kwargs['user_id'],
+            kwargs['course_id'],
+            kwargs['usage_id'],
```

Comment:
I'm not sure we *can* correlate that information accurately at the grades level.  All we have is the score for each submission, and there could well be two submissions in flight with the same score.  But I've added what we have to the log statement, so we can make a best-effort guess.